README.md

```diff
@@ -227,6 +284,7 @@ Alternatively, you can run the above steps (apart from `02_qa_generation.py`) us
     "job_description": "You may provide a description for the current job run which describes in words what you are about to experiment with",
     "sampling": {
         "sample_data": "Set to true to enable sampling",
+        "only_run_sampling": "If set to true, this will only run the sampling step and will not create an index or any subsequent steps, use this if you want to build a small sampled dataset to run in AML",
```

Comment:
Looks like a complicated way to configure the intent "stop after this step". Shall we rather have some config specifying a list of steps to make or the step name to stop on?

Reply:
Good suggestion: Would make sense when we refactor to be more composable and beyond the scope of this PR